tests/foreman/api/test_contentviewversion.py

```diff
@@ -160,6 +166,8 @@ def test_positive_promote_out_of_sequence_environment(self):
         :expectedresults: The promotion succeeds.
 
         :CaseLevel: Integration
+
+        :CaseImportance: High
```

Comment:
Same for this file as well https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentviewversion.py#L13